frontend/src/actions/telcoActions.js

```diff
@@ -197,18 +198,14 @@ export const getTelcoSummary = (countObj) => (dispatch) => {
 };
 
 export const setTelcoOtherSummaryFilter = () => (dispatch, getState) => {
-  const filteredResults = [...getState().telco.filteredResults];
-  const keyWordArr = ["success", "failure"];
-  const data = filteredResults.filter(
-    (item) => !keyWordArr.includes(item.jobStatus?.toLowerCase())
-  );
-  dispatch({
-    type: TYPES.SET_TELCO_FILTERED_DATA,
-    payload: data,
-  });
-  dispatch(tableReCalcValues());
+  const filterData = [...getState().telco.filterData];
+  const summary = getState().telco.summary;
```

Comment:
I believe we can further factor out the shared implementation details between all the `set<Perf Domain>OtherSummaryFilter()` by using bracket notation to dynamically set the performance domain.

```js
export const setTelcoOtherSummaryFilter = () => (dispatch, getState, perfDomain) => {
  const filterData = [...getState()[perfDomain].filterData];
  const summary = getState()[perfDomain].summary;

  if (summary?.othersCount !== 0) {
    const filteredStatus = filterOtherStatus(filterData);
    dispatch(setSelectedFilter("jobStatus", filteredStatus, false));
    dispatch(setAppliedFilters(perfDomain));
  }
```

hm, `setTelcoAppliedFilters()` and friends would need to be refactored as well.